Digitall.Warehouse.Application/Behaviors/LoggingMessageDefinitions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.Extensions.Logging;
+
+namespace Digitall.Warehouse.Application.Behaviors
+{
+    internal static partial class LoggerMessageDefinitionsGen
+    {
+        [LoggerMessage(EventId = 0, Level = LogLevel.Information, Message = "Handling feature '@{FeatureName}' with request '@{Request}'.")]
+        public static partial void LogHandlingFeatureRequestMessage(this ILogger logger, string featureName, object request);
+
+        [LoggerMessage(EventId = 0, Level = LogLevel.Information, Message = "Feature '@{FeatureName}' handled.")]
```

Comment:
The idea here is to have also different EventId for each different log message

Reply:
I've understood from the video that the `EventId` matters to differentiate the messages per resource for example. I've set it to 0 so this will be the generic exception handling logging... 
Here is an example:
```
public class MyLogEvents
{
    public const int GenerateItems = 1000;
    public const int ListItems     = 1001;
    public const int GetItem       = 1002;
    public const int InsertItem    = 1003;
    public const int UpdateItem    = 1004;
    public const int DeleteItem    = 1005;

    public const int TestItem  = 3000;

    public const int GetItemNotFound    = 4000;
    public const int UpdateItemNotFound = 4001;
}
```
In our case will be good if we can have event id vs command/query, which at this stage we do not have as implementation :( so for me was hard to choose how to generate the event id so I left it to 0.
Do you have any recommendations @georgiev84?
I'm thinking for smart enum where the command/query to be the id and the EvetId to be specified as number, but wonder if this will bring any benefit. We do log the name of the feature anyway.